Content.Server/Chemistry/EntitySystems/ChemistryGuideDataSystem.cs

```diff
@@ -26,12 +29,26 @@ public override void Initialize()
 
     private void InitializeServerRegistry()
     {
-        var changeset = new ReagentGuideChangeset(new Dictionary<string, ReagentGuideEntry>(), new HashSet<string>());
+        var changeset = new ReagentGuideChangeset(new Dictionary<string, ReagentGuideEntry>(), new HashSet<string>(), new Dictionary<string, Dictionary<string, uint>>(), new HashSet<string>());
```

Comment:
If these are just being initialized to default values there's no point having them in the ctor.